libafl/src/stages/tracing.rs

```diff
@@ -182,7 +212,16 @@ where
     /// Creates a new default stage
     pub fn new(_executor: &mut ShadowExecutor<E, SOT>) -> Self {
         Self {
+            initial_tries: 10,
             phantom: PhantomData,
         }
     }
+
+    /// Apply a certain amount of tries that this stage will attempt to trace the input before
+    /// giving up and not processing the input again.
+    #[must_use]
+    pub fn with_tries(mut self, initial_tries: usize) -> Self {
```

Comment:
_re_tries?

Reply:
`with_retries(1)` would suggest that the input is run twice, not once like current behaviour.

Reply:
So, you would prefer that the value represents the number of retries? In other words, 0 retries = 1 try?

Reply:
I've just pushed this change.